packages/adapters/solana/src/client.ts

```diff
@@ -659,9 +649,7 @@ export class SolanaAdapter implements ChainAdapter {
 
     for (const provider of providers) {
       this.availableProviders = this.availableProviders.filter(p => p.name !== provider.name)
-      if (provider.type !== 'WALLET_CONNECT') {
-        this.availableProviders.push(provider)
-      }
+      this.availableProviders.push(provider)
```

Comment:
is this ok?

Reply:
yes, all kinds of providers should be in the available providers, otherwise we would need to treat them differently losing the purpose of the abstraction.